src/__tests__/RadioButton.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { shallow } from 'enzyme'
+import { RadioButton, theme } from '..'
+
+describe('RadioButton', () => {
+  test('Selected, rendering', () => {
+    const json = renderer
+      .create(
+        <RadioButton
+          theme={theme}
+          isSelected
+          radioButtonText="Option Selected"
+        />
+      )
+      .toJSON()
+    expect(json).toMatchSnapshot()
+  })
+
+  test('Not Selected, rendering', () => {
+    const json = renderer
+      .create(
+        <RadioButton theme={theme} radioButtonText="Option Not Selected" />
+      )
+      .toJSON()
+    expect(json).toMatchSnapshot()
+  })
+
+  test('Disabled, rendering', () => {
+    const json = renderer
+      .create(
+        <RadioButton theme={theme} disabled radioButtonText="Option Disabled" />
+      )
+      .toJSON()
+    expect(json).toMatchSnapshot()
+  })
+})
+
+describe('RadioButton onClick', () => {
```

Comment:
Using the built-in React form attributes means you don't have to test custom handlers, so this could be cut